primer/brand
# Updates to grouped FAQs

## Summary

Updates appearance for displaying grouped FAQ content, to match recommendations from Site Refactor designs.

🔗 Storybook examples
🔗 Documentation

## List of notable changes:



- **added** a new `FAQGroup` component to wrap individual `FAQ` children. 
  - Aligns with the approach used for `ButtonGroup`  
- **updated** `Accordion` to support new `variant` prop, which now supports: `default` and `emphasis` formats that have unique visuals
- **updated** Accordion toggles to be end aligned by default, and optionally start align using a `reversedToggles` prop
- **updated** FAQ subheading styles (smaller and muted)
- **fixed** nested accordions, which previously didn't work


## What should reviewers focus on?

- Try it out in Storybook and Documentation
- Providing feedback on the new API

## Steps to test:



1. Go to this Storybook story
1. Check the design matches the proposal in light and dark mode

## Supporting resources (related issues, external links, etc):

- [Figma design file](https://www.figma.com/file/wuhr1sqnv2YEaT4F6dUBbl/2023-Refactor?type=design&node-id=2821-7334&mode=design&t=RGTqzmgHu35b1JwJ-4)

## Contributor checklist:


## Reviewer checklist:


## Screenshots:

> Please try to provide before and after screenshots or videos

<table>
<tr>
<th> Light</th> <th> Dark </th>
</tr>
<tr>
<td valign="top">

![Screenshot 2023-08-30 at 10 38 52](https://github.com/primer/brand/assets/13340707/5c6f7e89-b764-4134-83a9-511e7708a8b8)


 </td>
<td valign="top">

![Screenshot 2023-08-30 at 10 39 00](https://github.com/primer/brand/assets/13340707/05655258-a9d2-4ff1-aaa2-8626740316b9)


</td>
</tr>
</table>

**Narrow viewport presentation:**
![Screenshot 2023-08-30 at 10 39 37](https://github.com/primer/brand/assets/13340707/cdce992b-5b7d-4b8c-81af-90c45f7e99b5)
